code/game/objects/items/storage/backpack.dm

```diff
@@ -11,63 +11,31 @@
 	max_w_class = 3
 	storage_slots = null
 	max_storage_space = 24
-	var/worn_accessible = FALSE //whether you can access its content while worn on the back
-
-/obj/item/storage/backpack/attack_hand(mob/living/user)
-	if(!worn_accessible && ishuman(user))
-		var/mob/living/carbon/human/H = user
-		if(H.back == src)
-/*			if(user.dropItemToGround(src))
-				pickup(user)
-				if(!user.put_in_active_hand(src))
-					dropped(user)
-*/
-			to_chat(H, "<span class='notice'>You can't look in [src] while it's on your back.</span>")
-			return TRUE
-	return ..()
-
+	access_delay = 2.5 SECONDS
```

Comment:
It's either a boolean or a number of tick, not both

Reply:
It's the number of ticks. `FALSE` is equivalent to `0` so I use it.

Reply:
Don't use FALSE, use 0. It's hella confusing